Documentation/operations/upgrade.rst

```diff
@@ -333,6 +333,12 @@ Annotations:
   Egress rules in CiliumNetworkPolicy CRD. The old attribute name is no longer supported,
   please update your CiliumNetworkPolicy CRD accordingly. Also applicable values for this
   attribute are changed to ``disabled``, ``required`` and ``test-always-fail``.
+* Deny policies are now stable because of a fix to Cilium's internal CIDR policy
+  enforcement logic. Previously, a CIDR-based allow-policy would always allow traffic,
+  even if there was CIDR-based deny-policy in place that should have overridden it. Please
+  check all of your CIDR-based deny and allow policies to ensure that your cluster is not
+  accidentally relying on an allow-policy that should not work and will no longer work
+  after upgrading to 1.14.
```

Comment:
I agree with @joestringer that users need more information about how specifically to verify that their allow/deny policies work as intended. If there's content that already provides this, let's link to it here. If not, let's add that content and determine where the best place for it to live might be, whether on this page or in the upgrade guide linked by Joe. My preference is for the latter.

Edits for clarity, hyphen nit, use clear, plain language
```suggestion
* Due to a fix to Cilium's internal CIDR policy enforcement logic in v1.14, deny policies are now stable.
  Previously, a CIDR-based allow policy allowed all traffic,
  even if there was a CIDR-based deny policy to deny traffic.
  
  Verify that all of your CIDR-based deny and allow policies work as intended, so that your cluster does not
  accidentally rely on an overly permissive allow policy that will not work
  after upgrading to 1.14.
```